README.md

```diff
@@ -68,7 +68,7 @@ Use the [os-maven-plugin](https://github.com/trustin/os-maven-plugin) to add the
 <dependency>
   <groupId>org.conscrypt</groupId>
   <artifactId>conscrypt-openjdk</artifactId>
-  <version>2.5.0</version>
+  <version>2.5.1</version>
```

Comment:
I'm not familiar with maven or gradle but I wonder if there is a way to specify "latest" or equivalent so we don't have to keep updating these notes and also don't encourage users to hardcode (eventually) old version of the dependency

Reply:
For consumers of Conscrypt it depends on the tools they're using, e.g. Android Studio will highlight out-of-date dependencies in Gradle files and offer to update them, but I don't think either Gradle or Maven have a way to express "always use the latest available) but I could be wrong.

For us keeping the README up to date, it would be nice if there were a way to interpolate variables into Markdown but there isn't.  There is an explicit step in our release docs to update the README but I failed to follow it.